File: android/Sample01/app/src/main/java/com/naver/sample01/MainActivity.java
package com.naver.sample01;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    Button btn1;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        btn1 = (Button)findViewById(R.id.button1);
        btn1.setOnClickListener(new View.OnClickListener(){
            @Override
            public void onClick(View v) {
                Toast.makeText(getApplicationContext(), "삭제되었습니다.123",Toast.LENGTH_LONG).show();
            }
        });
    }
}

File: android/FragmentStudy/app/src/main/java/wikibook/learnandroid/fragmentstudy/MainActivity.kt
package wikibook.learnandroid.fragmentstudy

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.util.Log
import android.widget.Toast

// (1)
class MainActivity : AppCompatActivity(), CurrencyConverterFragment3.CurrencyCalculationListener {
    // (2)
    override fun onCalculate(result: Double, amount: Double, from: String, to: String) {
        Log.d("this", "${String.format("%.5f",amount)}($from) -> ${String.format("%.5f",result)}($to)")
        Toast.makeText(this, "${String.format("%.5f",amount)}($from) -> ${String.format("%.5f",result)}($to)", Toast.LENGTH_LONG).show()
    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val transaction = supportFragmentManager.beginTransaction()
        // (3)
        transaction.add(R.id.fragment_container, CurrencyConverterFragment3.newInstance("KRW", "USD"))
        transaction.add(R.id.fragment_container, CurrencyConverterFragment3.newInstance("JPY", "KRW"))
        transaction.add(R.id.fragment_container, CurrencyConverterFragment3.newInstance("EUR", "JPY"))

        transaction.commit()
    }
}

